### PR TITLE
[TG-TECH][refactor] Removed duplicated package

### PR DESCRIPTION
**Pull request by Appius**
Removed duplicated package - it's already declared on the line 67.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/121" target="_blank">ReviewNinja

File: package.json [hunk 1]
```diff
@@ -54,7 +54,6 @@
     "@angular/router": "3.0.0",
     "@angularclass/request-idle-callback": "^1.0.7",
     "@angularclass/webpack-toolkit": "^1.3.3",
-    "@types/codemirror": "0.0.33",
```
Comment (piton13) [inline package.json]: !fix asdf

File: package.json [hunk 2]
```diff
@@ -54,7 +54,6 @@
...
     "codemirror": "^5.18.2",
     "core-js": "^2.4.1",
     "dragula": "^3.7.1",
```
Comment (piton13) [inline package.json]: asdffdas

File: package.json [hunk 3]
```diff
@@ -54,7 +54,6 @@
     "@angular/router": "3.0.0",
     "@angularclass/request-idle-callback": "^1.0.7",
```
Comment (Appius) [inline package.json]: resolved

Comment (Appius) [inline package.json]: fixed

Comment (Appius) [inline package.json]: !resolved

File: package.json [hunk 4]
```diff
@@ -54,7 +54,6 @@
...
     "@angularclass/webpack-toolkit": "^1.3.3",
-    "@types/codemirror": "0.0.33",
     "codemirror": "^5.18.2",
```
Comment (piton13) [inline package.json]: !fix it

File: package.json [hunk 5]
```diff
@@ -54,7 +54,6 @@
...
     "core-js": "^2.4.1",
```
Comment (piton13) [inline package.json]: !completed asdf

File: package.json (hunk 2 above)
Comment (piton13) [inline package.json]: !fixed fdsa

**Pull request merged**